docs/override-configs.md

```diff
@@ -1,3 +1,29 @@
+## Enabling/disabling components
+
+To enable/disable components in Ozone, you need to modify the configurations in the [docker-compose-files.txt](https://github.com/ozone-his/ozone-docker-compose/blob/main/scripts/docker-compose-files.txt) in the ozone-docker-compose for your Ozone deployment. Here's how you can do it:
+
+**Access the excutable files**
+
+Navigate to the docker-compose-files.txt file in the ozone-docker-compose.
```

Comment:
Not sure what that means.
> in the ozone-docker-compose.

Are you talking about the repo? A folder?

Actually, in the case of a child distribution of Ozone, one must create a new file (in the `scripts/` folder) and I think add a build step in the pom to override the file provided by the Ozone Docker Compose with the one provided locally.